openwisp_utils/admin_theme/admin.py

```diff
@@ -3,20 +3,27 @@
 from django.utils.translation import ugettext_lazy
 
 
-def openwisp_admin(site_url=None):
+class OpenwispAdminSite(admin.AdminSite):
     # <title>
-    admin.site.site_title = getattr(settings,
-                                    'OPENWISP_ADMIN_SITE_TITLE',
-                                    'OpenWISP Admin')
-    # link to frontend
-    admin.site.site_url = site_url
+    site_title = getattr(settings,
+                         'OPENWISP_ADMIN_SITE_TITLE',
+                         'OpenWISP Admin')
     # h1 text
-    admin.site.site_header = getattr(settings,
-                                     'OPENWISP_ADMIN_SITE_HEADER',
-                                     'OpenWISP')
+    site_header = getattr(settings,
+                          'OPENWISP_ADMIN_SITE_HEADER',
+                          'OpenWISP')
     # text at the top of the admin index page
-    admin.site.index_title = ugettext_lazy(
+    index_title = ugettext_lazy(
         getattr(settings,
                 'OPENWISP_ADMIN_INDEX_TITLE',
                 'Network administration')
     )
+
+
+# openwisp_admin function is deprecated and discouraged to use,
+# you should use admin_site object from openwisp_utils.admin_theme.site instead
+def openwisp_admin(site_url=None):
```

Comment:
Small nitpick, since federico commented: "Add a docstring to the function explaining that its usage it's deprecated."

Can you please change this to:
```
def openwisp_admin(site_url=None):
"""
openwisp_admin function is deprecated and discouraged to use,
you should use admin_site object from openwisp_utils.admin_theme.site instead
"""
``` 

docstrings and comments have a small differences, some documenation parsers read the docstring to explain the user important notice / usage about the function.
I am not sure if that'll be important in this case but can you still make this change?